packages/preset-create-react-app/src/helpers/mergePlugins.ts

```diff
@@ -11,16 +11,16 @@ export const mergePlugins = (...args: Plugin[]): Plugin[] =>
     ) {
       return plugins;
     }
-    let updatedPlugin = plugin;
-    if (plugin.constructor.name === 'ReactRefreshPlugin') {
-      // Storybook uses webpack-hot-middleware
-      // https://github.com/storybookjs/presets/issues/177
+    const updatedPlugin = plugin;
+    // if (plugin.constructor.name === 'ReactRefreshPlugin') {
+    //   // Storybook uses webpack-hot-middleware
+    //   // https://github.com/storybookjs/presets/issues/177
 
-      updatedPlugin = new ReactRefreshWebpackPlugin({
-        overlay: {
-          sockIntegration: 'whm',
-        },
-      });
-    }
+    //   updatedPlugin = new ReactRefreshWebpackPlugin({
+    //     overlay: {
+    //       sockIntegration: 'whm',
+    //     },
+    //   });
+    // }
```

Comment:
I'm not sure about this @shilman. We could just remove it, but I don't remember why this was added.